cpp/job/tests.cpp

```diff
@@ -3,33 +3,40 @@
 #include "job_manager.h"
 #include "../log/log.h"
 
+#include <thread>
+#include <chrono>
+
 namespace openage {
 namespace job {
 namespace tests {
 
 
-int test_simple_job() {
+int test_simple_job () {
 	JobManager manager{4};
 	manager.start();
 
-	bool finished = false;
+	std::atomic<int> finish_count(0);
+	int job_count = 10;
 	bool result = -1;
 
 	auto job_function = []() -> int {
+		std::this_thread::sleep_for(std::chrono::milliseconds(1));
```

Comment:
nice you lowered it. but thinking about it, what is the reason you need to add any delay here?